packages/base/Form/src/FieldRequirements/FieldRequirements.tsx

```diff
@@ -49,42 +49,44 @@ export const FieldRequirements = <TValueType,>({
       timeout={timeout}
       data-testid={testIds?.root}
     >
-      {description && (
-        <Typography
-          data-testid={testIds?.description}
-          variant='body'
-          size='xxsmall'
-          className='!mt-[0.4rem]'
+      <>
+        {description && (
+          <Typography
+            data-testid={testIds?.description}
+            variant='body'
+            size='xxsmall'
+            className='!mt-[0.4rem]'
+          >
+            {description}
+          </Typography>
+        )}
+        <Grid
+          className='w-input'
+          spacing={0}
+          data-testid={testIds?.gridContainer}
         >
-          {description}
-        </Typography>
-      )}
-      <Grid
-        className='w-input'
-        spacing={0}
-        data-testid={testIds?.gridContainer}
-      >
-        {requirements.map(requirement => {
-          let status: FieldRequirementItemStatus = 'default'
+          {requirements.map(requirement => {
+            let status: FieldRequirementItemStatus = 'default'
 
-          // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-          if (requirement.validator(value!)) {
-            status = 'success'
-          } else if (error) {
-            status = 'error'
-          }
+            // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+            if (requirement.validator(value!)) {
```

Comment:
I would also remove this ignore comment line for a real check, no need for ignoring a eslint rule for such a simple conditional check
```suggestion
            if (value != null && requirement.validator(value)) {
```
The code was already this way, but as in [the boy scout rule](https://martinfowler.com/bliki/OpportunisticRefactoring.html), we might squeeze an improvement here, wdyt?